### PR TITLE
Invert payment type flag logic and fix a few bugs

### PR DESCRIPTION
**Pull request by remybach**
## Both disabled

Before:

After:

## Direct Debit disabled

Before:

After:

## Paypal disabled

Before:

After:

### CODE REVIEW

File: views/partials/enhanced-experience.html [hunk 1]
```diff
@@ -42,18 +42,18 @@ <h1 class="su-header__title"><span class="su-header__title--light">Subscribe to<
 			</div>
 			{{/if}}
 
-			{{#ifSome @root.flags.paypal @root.flags.directDebit}}
+			{{#if noPaypalOrDD}}
```
Comment (geek-caroline) [inline views/partials/enhanced-experience.html]: Don't know enough about it to comment, but poss worth putting in the ft-next-dev chat that you'd like to add it to n-handlebars?

File: server/routes/signup/controller.js [hunk 2]
```diff
@@ -220,6 +220,10 @@ export default function (req, res, next) {
 		view.model.contentId = req.query['ft-content-uuid'];
 		view.model.useBetaSignupApiEndpoints = res.locals.flags['nextSubscriptionAppWithBetaSignupApi'];
 		view.model.useTestEnvironment = res.locals.flags['nextSubscriptionAppWithTestApis'];
+
+		view.model.showPaypal = !res.locals.flags['nextSignupFormDisablePaypal'];
+		view.model.showDD = !res.locals.flags['nextSignupFormDisableDD'];
+		view.model.noPaypalOrDD = !view.model.showPaypal || !view.model.showDD;
```
Comment (geek-caroline) [inline server/routes/signup/controller.js]: Should this be `&&` or am I being dense?

File: views/partials/enhanced-experience.html (hunk 1 above)
Comment (remybach) [inline views/partials/enhanced-experience.html]: Good call... I've asked there 👍

File: server/routes/signup/controller.js (hunk 2 above)
Comment (remybach) [inline server/routes/signup/controller.js]: Dealing with double negatives has been bending my brain a bit working on this, but it's basically the same thing. I'll update it to `&&` as it's clearer 👍

**Pull request merged**